.github/ISSUE_TEMPLATE/default.md

```diff
@@ -0,0 +1,20 @@
+---
+name: default
+about: Default Issue Template
+title: ''
```

Comment:
lack of knowledge question: Is this front matter? what's the difference between these fields and using the options on GitHub PR UI?

Reply:
I think you can make a title prefix or add default labels. We can always adjust as needed